core/tauri/src/app.rs

```diff
@@ -1753,6 +1753,76 @@ tauri::Builder::default()
   }
 }
 
+// enhance core `.plugin()` for mobile and desktop
+impl<R: Runtime> Builder<R> {
+  /// same behavior like `.plugin()` but skips on non desktop builds and ignores `target_os`
```

Comment:
Let's change all these docs to start with capital
```suggestion
  /// Same behavior like `.plugin()` but skips on non desktop builds and ignores `target_os`
```